Source/MediaInfo/Audio/File_Usac_Data.h

```diff
@@ -0,0 +1,350 @@
+/*  Copyright (c) MediaArea.net SARL. All Rights Reserved.
+ *
+ *  Use of this source code is governed by a BSD-style license that can
+ *  be found in the License.html file in the root of the source tree.
+ */
+
+
+//---------------------------------------------------------------------------
+#ifndef MediaInfo_File_Usac_DataH
+#define MediaInfo_File_Usac_DataH
+//---------------------------------------------------------------------------
+
+
+//---------------------------------------------------------------------------
+
+namespace MediaInfoLib
+{
+
+//---------------------------------------------------------------------------
+static const int16u huffman_scl[][4]=

```

Comment:
Not static so we can catch when it is included more than once. Wondering of not better to put them directly in the .cpp.